authzed/api/v1/debug.proto

```diff
@@ -23,6 +23,9 @@ message DebugInformation {
 
   // schema_used holds the schema used for the request.
   string schema_used = 2;
+
+  // checks holds debug information about the individual checks made in a bulk check request.
+  repeated CheckDebugTrace checks = 3;
```

Comment:
Do we need to deprecate the `check` field so that we don't have both `check` and `checks`? It seems confusing.

Reply:
We could; we only need the checks field when we turn a batch check into a set of chunks, but we could (for now) just fill in both and switch `zed` to always use the `checks` one, to ensure back-compat